docs/operating-eck/installing-eck.asciidoc

```diff
@@ -17,9 +17,9 @@ CAUTION: Deleting CRDs will trigger deletion of all custom resources ({eck_resou
 
 
 [id="{p}-install-all-in-one"]
```

Comment:
Nit: rename the anchor?
```suggestion
[id="{p}-install-yaml-manifests"]
```

Reply:
I guess I was hesitant to change this one because I thought there might be external references to that. But I guess we only link to the quickstart from the the download page (which we need to update for the next release)